keyboards/plx/readme.md

```diff
@@ -0,0 +1,17 @@
+# PLX
+
+![PLX](https://i.imgur.com/xJvv1kq.jpg)
+
+An 60% gasket mounted board with a Tsangan layout and a 7 degree typing angle.
+
+* Keyboard Maintainer: [2Moons](https://github.com/2Moons-JP)  
+* Hardware Supported: PLX PCB, ATMEGA32U4-AU
+* Hardware Availability: [Website](https://peac.design)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?  Eg, is there a physical reset button? etc.